mappings/net/minecraft/world/gen/feature/CountFeatureConfig.mapping

```diff
@@ -0,0 +1,8 @@
+CLASS net/minecraft/class_2984 net/minecraft/world/gen/feature/CountFeatureConfig
+	FIELD field_13385 count Lnet/minecraft/class_5428;
+	FIELD field_24878 CODEC Lcom/mojang/serialization/Codec;
+	METHOD <init> (I)V
+		ARG 1 count
+	METHOD <init> (Lnet/minecraft/class_5428;)V
+		ARG 1 count
+	METHOD method_30396 getCount ()Lnet/minecraft/class_5428;
```

Comment:
This class has already be mapped and is named `CountConfig`

Reply:
Yeah, I haven't yet merged this with master properly.